reana_workflow_engine_serial/config.py

```diff
@@ -28,3 +28,6 @@
 
 MOUNT_CVMFS = True if os.getenv('REANA_MOUNT_CVMFS').lower() == 'true' \
     else False
+
+JOB_STATUS_POLLING_INTERVAL = 1
```

Comment:
Hmm,1 might be perhaps too frequent for production deployment conditions? We could start introducing environment variables when appropriate so that people could alter the behaviour easily on their deployments.